### ISSUE TITLE
Why are both prompts and config required?

### ISSUE THREAD

**Issue by nwaughachukwuma** [promptfoo/promptfoo-action#107]

**Comment by typpo** [comment]
@mocyuto would it make sense to make `prompts` optional?

**Comment by nwaughachukwuma** [comment]
Yeah, since the config can be used to provide a reference to the prompt

**Comment by nwaughachukwuma** [comment]
@typpo, promptfoo allows specifying the prompt pattern in the config file. Is there any reason for making this parameter required?

**Comment by nwaughachukwuma** [comment]
> https://github.com/promptfoo/promptfoo-action/pull/108/files

I see there's a PR in the works.

**Comment by dorperetz-fiverr** [comment]
would love this PR to be merged as well.

**Comment by polmann** [comment]
it still errors when I don't provide a prompts config, is this released?

**Comment by hwaxxer** [comment]
This issue was created 13 months ago and prompts are still required... I made a PR #414 but this repo is practically dead. The only commits are automatic package updates.
